tests/test_client.py

```diff
@@ -12,6 +14,8 @@
     from urllib import request as http
     http_urlopen = 'urllib.request.urlopen'
 
+http_urlopen = 'requests.Session.send'
+
```

Comment:
Why are we doing this, this way again?

Reply:
We are using http_urlopen in test_http_request in order to use @patch to mock a Response

Reply:
oh okay :face_palm:  I might fix up some of this in future PRs